packages/-build-infra/bin/test-external-partner-project.js

```diff
@@ -32,31 +33,90 @@ console.log(
 );
 
 function execExternal(command, force) {
-  command = `cd ${cachePath}/${externalProjectName} && ${command}`;
+  command = `cd ${projectTempDir} && ${command}`;
   return execWithLog(command, force);
 }
 
 function execWithLog(command, force) {
+  debug(chalk.cyan('Executing: ') + chalk.yellow(command));
   if (debug.enabled || force) {
     return shellSync(command, { stdio: [0, 1, 2] });
   }
 
   return shellSync(command);
 }
 
+function generatePackageConnectionCommand(useYarn, shouldLink = true) {
+  const packageManager = useYarn ? 'yarn' : 'npm';
+  const command = shouldLink ? `${packageManager} link` : `${packageManager} unlink`;
+  const packageConnectionCommand = `
+cd ${projectRoot};
+${shouldLink ? (useYarn ? 'yarn install --force;' : 'npm install;') : ''}
```

Comment:
Why do we need force for yarn?

Reply:
If you've just unlinked a package yarn doesn't understand it needs to replace it.

Reply:
(This is in yarn's own guidance for what to do following unlink)